cluster/ci/config/prow/openshift/deck.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: deck-prow
+  annotations:
+    "openshift.io/display-name": deck-prow
+parameters:
+- description: The namespace to get the image from.
+  name: IMAGE_NAMESPACE
+  value: prow-images
+- description: The namespace to instantiate deck under.
+  name: NAMESPACE
+  value: ci
+- description: The name of the component.
+  name: NAME
+  value: deck
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: "${NAME}"
+    namespace: "${NAMESPACE}"
+  spec:
+    selector:
+      app: "${NAME}"
```

Comment:
Can we do similarly -- Prow is all one app, deck uses some other label (`component`?) to differentiate? Otherwise our UI overview is going to be a mess